packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -307,13 +308,9 @@ contract Staker is StakerInterface, Ownable {
      ****************************************/
 
     // Determine if we are in an active reveal phase. This function should be overridden by the child contract.
-    function inActiveReveal() internal virtual returns (bool) {
-        return false;
-    }
+    function inActiveReveal() internal view virtual returns (bool) {}
 
-    function getStartingIndexForStaker() internal virtual returns (uint64) {
-        return 0;
-    }
+    function getStartingIndexForStaker() internal view virtual returns (uint64) {}
```

Comment:
nit: I think changes from https://github.com/UMAprotocol/protocol/pull/4068 were accidentally included. We should probably drop this commit.